lib/web/integrations_awsoidc.go

```diff
@@ -871,6 +871,54 @@ func (h *Handler) awsOIDCDeployEC2ICE(w http.ResponseWriter, r *http.Request, p
 	}, nil
 }
 
+// awsOIDCDeployC2ICE creates an AppServer that uses an AWS OIDC Integration for proxying access.
+func (h *Handler) awsOIDCCreateAWSAppAccess(w http.ResponseWriter, r *http.Request, p httprouter.Params, sctx *SessionContext, site reversetunnelclient.RemoteSite) (any, error) {
+	ctx := r.Context()
+
+	integrationName := p.ByName("name")
+	if integrationName == "" {
+		return nil, trace.BadParameter("an integration name is required")
+	}
+
+	appServer, err := types.NewAppServerForAWSOIDCIntegration(integrationName, h.cfg.HostUUID)
```

Comment:
It is not a problem if this app server already exists? I mean since we upsert it shouldn't matter, just checking that there's no logical problem or something.

Reply:
Yeah, I don't think there's an issue with upserting.